rustls/src/server/tls13.rs

```diff
@@ -296,6 +300,7 @@ mod client_hello {
             let (mut ocsp_response, mut sct_list) =
                 (server_key.get_ocsp(), server_key.get_sct_list());
             emit_encrypted_extensions(
+                &self.config,
```

Comment:
Should `config` parameters be at the beginning or end of the argument list? Let's be consistent.

Reply:
Argument order here is a bit of mess anyway, moved it to the end.